import/popups/PopupOpener.qml

```diff
@@ -34,7 +34,8 @@ Timer {
         "embed:auth":           "authPopup",
         "embed:permissions":  { "geolocation": "locationPermissionPopup" },
         "embed:webrtcrequest":  "webrtcPermissionPopup",
-        "embed:popupblocked":   "blockedTabPopup"
+        "embed:popupblocked":   "blockedTabPopup",
+        "embed:select":         "selectorPopup"
```

Comment:
How did you ended up resurrecting long time ago deprecated embed:select?

One reason would well be that it got dropped quite recently:
https://github.com/sailfishos/embedlite-components/commit/c0f7f03ef39db0bb915925284f7c696cf7370703

There's "embed:selectasync" in PickerOpener for handling \<select\> (single and multi).

Reply:
I'll resolve this comment as per the discussion on the related changes in embedlite-components: https://github.com/sailfishos/embedlite-components/pull/5#discussion_r667088133